html.md

```diff
@@ -260,3 +260,34 @@ actual filename.
 
 Whenever possible, use actual text instead of images of text, styling the text
```

Comment:
Most of the references here are singular, except for "images of text". logo, "actual text", etc.

I think changing "images of text" to "an image of text" makes everything the same singular format.

I know that is what was here before, and it was just being carried through, but this might be the time to fix that.

`Whenever possible, use actual text instead of an image of text...`

`In situations where you must use an image of text instead of actual text (for example, a logo)...`

Reply:
Nice catch! Fixed those! Thanks!